docs/UserGuide.md

```diff
@@ -2,10 +2,13 @@
 UNOFAS
 ---
 
-UNOFAS (One Financial Advisors app) is a **desktop app for Financial Advisors to manage client's contacts,
+UNOFAS (UNO: One FAS: Financial Advisors app) is a **desktop app for Financial Advisors to manage client's contacts,
 optimized for use via a Command Line Interface** (CLI) while still having the benefits of a
 Graphical User Interface (GUI). If you can type fast, UNOFAS can help you manage and retrieve client's information
-better than traditional GUI apps.
+better than traditional GUI apps. 
+
+This guide provides you with comprehensive instructions on utilizing UNOFAS. It also serves as a mode of referral to help learn the various commands required
+to effectively learn and be able to integrate the application to your daily use as financial advisors.
```

Comment:
We can include a short description of how user can navigate through guide based on rubrics.